docs/config/server-options.md

```diff
@@ -161,7 +161,7 @@ export default defineConfig({
 ## server.cors
 
 - **Type:** `boolean | CorsOptions`
-- **Default:** `false`
+- **Default:** `{ origin: /^https?:\/\/(?:(?:[^:]+\.)?localhost|127\.0\.0\.1|\[::1\])(?::\d+)?$/ }` (allows localhost, `127.0.0.1` and `::1`)
```

Comment:
Maybe we can also show that you can use `defaultAllowedOrigins` here so users don't copy the regex as is

Reply:
I didn't expose `defaultAllowedOrigins` so it's not possible to import it.

Reply:
Ah didn't notice that. I think it'd be useful to expose it, but we can do that in a later PR.